docs/primitives/a-circle.md

```diff
@@ -6,8 +6,7 @@ parent_section: primitives
 source_code: src/extras/primitives/primitives/meshPrimitives.js
 ---
 
-The circle primitive creates circles surfaces using the [geometry][geometry]
-component with the type set to `circle`.
+The circle primitive creates a flat two-dimensional circle using the [geometry component] with type set to `circle`.
```

Comment:
Same as my previous comment about a-box. This description while technically correct is less colloquial and beginner friendly.